docs_devel/docs/93.BuildingInstallerPackage.md

```diff
@@ -158,27 +158,28 @@ The following packages are recognized as Linux JREs:
 
 OmegaT has tasks to generate Linux native packages in "DEB" and "RPM" archive formats.
 We use the `jpackage` packaging tool which is provided by Java Development Kits.
-The tool has been introduced in Java 14. OmegaT project requires JDK 17.
+The tool was introduced in Java 14, which means the OmegaT project requires JDK 17
+to build packages.
 
 The packaging tool `jpackage` generates installable packages for Java applications.
-You can provide platform-specific packages for Linux, macOS and Windows to users that come
-with familiar ways to install and launch the packaged applications.
+You can provide platform-specific packages for Linux, macOS or Windows that can be
+installed and launched in ways familiar to user.
 
-`jpackage` bundles the java runtime which is used to run Gradle or the one which is used to compile the project
-that has the `jpackage` utility. Because OmegaT project uses Java 11 to compile, you need to
-run Gradle with JDK17.
--> Why ???
+`jpackage` bundles the java runtime used to run Gradle, but can also be run with the runtime
+used to compile the project that relies on the `jpackage` utility. Fpr OmegaT, you need to
+run Gradle with JDK 17 to create packages, because the application itself is compiled with
+Java 11, which is not compatible with `jpackage`.
 
-OmegaT uses the `gradle-jpackage-plugin` that integrate `jpackage` command line utility
-which is bundled in JDK with Gradle build utility.
+OmegaT uses the `gradle-jpackage-plugin`, which integrates the `jpackage` command line utility
+bundled in the JDK into the Gradle build tool.
 -> I do not understand this sentence
 
 You can use the following tasks to produce the packages.
 
 - `linuxDebDist`
 - `linuxRpmDist`
 
-You can debug the contents of the package by running the `jpackage` taks to produce the
+You can debug the contents of the package by running the `jpackage` tasks to produce the
```

Comment:
```suggestion
You can debug the contents of the package by running the `jpackage` task to produce the
```

The `jpackage` task, `$./gradlew jpackage`, generate directory structure in `build/jpacakge/app-image/OmegaT` like `app-image/OmegaT/bin/OmegaT`, `app-image/OmegaT/lib/app/OmegaT.jar` , `app-image/OmegaT/lib/app/languagetool-core-6.1.jar`, and so on. 

There are another `JPackage` type tasks, `distDeb` and `distRpm`,  generate final `deb` and `rpm` files.

Reply:
Got it. Thanks for the extra explanation.